concordium-sdk/src/main/java/com/concordium/sdk/transactions/InitName.java

```diff
@@ -0,0 +1,33 @@
+package com.concordium.sdk.transactions;
+
+import com.concordium.sdk.types.UInt16;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import lombok.Getter;
+import lombok.ToString;
+import lombok.val;
+
+import java.nio.ByteBuffer;
+
+
+@Getter
+@ToString
+public final class InitName {
```

Comment:
Documentation.

Reply:
So there's a few rules about how these init functions can look like. I think it would be an good idea to enforce that. 
It must start with a `init_` prefix. I.e. `init_<contract_name>`.
Also the type should enforce that the maximum allowed characters in the string is 100, 
It must only contain ascii alphanumeric or punctuation  (but not a '.')